dashboards/cockroach-db/cockroach-db.json

```diff
@@ -95,7 +95,7 @@
             "nrqlQueries": [
               {
                 "accountId": 0,
-                "query": "SELECT (average(`sys_fd_open_number`) * cardinality(`sys_fd_open_number`)) As 'Open(Avg)', sum(sys_fd_softlimit_number) AS 'Soft Limit' FROM (SELECT (average(sys_fd_open) * cardinality(sys_fd_open)) AS `sys_fd_open_number`,  (average(sys_fd_softlimit) * cardinality(sys_fd_softlimit)) AS `sys_fd_softlimit_number` FROM Metric WHERE (app_kubernetes_io_name LIKE '%cockroach%' or app_newrelic_com_name LIKE '%cockroach%') And (instrumentation.name='remote-write') FACET instance ) FACET instance"
+                "query": "SELECT (average(`sys_fd_open_number`) * cardinality(`sys_fd_open_number`)) As 'Open(Avg)', sum(sys_fd_softlimit_number) AS 'Soft Limit' FROM (SELECT (average(sys_fd_open) * cardinality(sys_fd_open)) AS `sys_fd_open_number`,  (average(sys_fd_softlimit) * cardinality(sys_fd_softlimit)) AS `sys_fd_softlimit_number` FROM Metric WHERE  (app_kubernetes_io_name LIKE '%cockroach%' or app_newrelic_com_name LIKE '%cockroach%' or k8s_app LIKE '%cockroach%') AND (instrumentation.name = 'remote-write') AND (prometheus_server IN ({{prometheus_server}})) AND (instance IN ({{instance}})) FACET ({{dimensions}}) ) FACET ({{dimensions}})"
```

Comment:
```
FACET ({{dimensions}}) ) FACET ({{dimensions}})
```

2 facets on this condition, please remove 1

Reply:
This is nested query and facets are part of respective queries and not for the same queries. Both facets are needed and same have been tested. 
Making no changes here.